include/mbgl/map/backend.hpp

```diff
@@ -20,6 +20,9 @@ class Backend {
     // Returns the backend's context which manages OpenGL state.
     gl::Context& getContext();
 
+    using glProc = void (*)();
+    void initializeExtensions(glProc (*getProcAddress)(const char*));
```

Comment:
I suggest inverting the control here: add an abstract method `glProc getExtensionProc(const char * name) = 0;`, to be called by `Context` at an appropriate time. That will clean up the `Backend` implementations and ensure that they can't forget to initialize the extensions.